test/Settings.ui.ts

```diff
@@ -100,20 +102,18 @@ describe('Settings', function () {
 
   describe('Export accounts', function () {
     it('creates an encrypted keystore file containing account details', async function () {
-      await this.driver
-        .wait(
-          until.elementLocated(By.css('[data-testid=exportMenuItem]')),
-          this.wait
-        )
-        .click();
-      await this.driver
-        .wait(
-          until.elementLocated(
-            By.css('input[value="3P5Xx9MFs8VchRjfLeocGFxXkZGknm38oq1"]')
-          ),
-          this.wait
-        )
-        .click();
+      const item = await this.driver.wait(
+        until.elementLocated(By.css('[data-testid=exportMenuItem]')),
+        this.wait
+      );
+      await item.click();
```

Comment:
Does it make a difference if you `await .wait` and then `await .click`? As far as I understand there should be no difference https://www.selenium.dev/selenium/docs/api/javascript/module/selenium-webdriver/lib/webdriver_exports_WebElementPromise.html